src/Connection.php

```diff
@@ -72,14 +72,19 @@ public function __call($name, array $arguments)
     {
         $this->connect();
 
-        if (! method_exists($this->pdo, $name)) {
+        /** @var PDO $pdo */
```

Comment:
I think you have to use mixins here instead. Check https://phpstan.org/writing-php-code/phpdocs-basics#mixins